docs/sources/developers/plugins/_index.md

```diff
@@ -2,81 +2,52 @@
 aliases:
   - ../plugins/developing/
 description: Resources for creating Grafana plugins
-title: Build a plugin
+title: Welcome to the Grafana plugin developer's guide
 weight: 200
 ---
 
-# Build a plugin
+# Grafana plugin developer's guide
 
-For more information on the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}).
+Grafana is an open source software project for creating, exploring, and sharing data through dashboards. Plugins extend Grafana's built-in functionality so that users can accomplish specialized tasks to make Grafana even more useful for them. 
 
-## Get started
+Many plugins are available in the [Catalog](https://grafana.com/grafana/plugins/) and others are built by developers for private use or to contribute them to the open source community. 
 
-The easiest way to start developing Grafana plugins is to use the Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin).
+If you are a Grafana plugin developer or want to become one, this documentation is for you. Use it to get productive quickly, follow along with step-by-step tutorials, or you can refer back to it as you develop.  
 
-Open the terminal, and run the following command in your [plugin directory]({{< relref "../../setup-grafana/configure-grafana/#plugins" >}}):
+## Plugin basics
 
-```bash
-npx @grafana/create-plugin@latest
-```
+These are among the types of plugins you can create:
 
-Follow the questions and you will have a starter plugin ready to develop.
+- **Panel plugins** - Visualize data and navigate between dashboards.
+- **Data source plugins** - Link to new databases or other sources of data.
+- **App plugins** - Bundle panels and data sources together to create a better user experience.
```

Comment:
"App Plugins" -> I think the description is not quite correct and I know we are using it somewhere on our website - e.g. https://grafana.com/grafana/plugins/

I would like to emphasise that it's not just bundling panels and datasources together but you can do way more. It is basically a custom react + backend application running inside Grafana and you are free to do whatever you need in there. Some examples of App Plugins Grafana Labs has built:
- https://play.grafana.org/a/k6-app
- https://play.grafana.org/a/grafana-incident-app/incidents
- https://play.grafana.org/a/grafana-ml-app/home
- https://play.grafana.org/a/grafana-k8s-app/cluster

 I would say something like

"**App plugins** - Create rich applications for custom out-of-the-box experience"

Reply:
Agree with the above